lib/plugins/visualize/visualize.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const chalk = require('chalk');
+const archy = require('archy');
+
+class Visualize {
```

Comment:
This needs test coverage as well

(But should (of course) be added after we talked about the details).